### PR TITLE
week 3 project: Business website

### PR DESCRIPTION
**Pull request by nehrwein**
# Business Site

This project is about building a responsive business website with a hero image or video for the header, a form and an accordion-FAQ (JS). I chanelled my inner buddha and decided to build a website for a meditation center. Namasté 

## View it live
https://nehrwein-project-business-site.netlify.app

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,7 +1,291 @@
...
 /* the + sign is used to select the sibling that comes immediately next to the element you selected.
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
 .active + .answer {
-  /* some styling here */
+  display: block;
+}
+
+.question.active::after {     /* changes the plussign after the question to a minus. That was a tricky one. Needed some time to realize that this method only works, if I apply it on the container, that the ::after is actually positioned in */
```
Comment (Rosanna86) [inline code/style.css:207]: Well done! Think it looks good. Be proud!

File: code/style.css [hunk 2]
```diff
@@ -1,7 +1,291 @@
...
+  display: flex;
+  align-items: flex-end;
+  text-align: justify;
+}
+
+
+.subscription-form, .accordion {
+  max-width: 375px;
+  height: 530px;          /* Form takes up only 80% of the height on mobile */
+  border-radius: 20px 20px 0 0;
```
Comment (Rosanna86) [inline code/style.css:90]: Nice with border radius on mobile mode!

File: code/index.html [hunk 3]
```diff
@@ -4,27 +4,153 @@
...
+        <label class="form-label">      <!-- Phone no. -->
+          <input
+            class="text-input" 
+            type="tel"
+            name="phone"
+            placeholder="Mobile Phone"
+            />
+        </label>
+
+        <fieldset class="experience-fieldset">
```
Comment (Rosanna86) [inline code/index.html:72]: Nice with the fieldset element

File: code/style.css [hunk 4]
```diff
@@ -1,7 +1,291 @@
...
+/* TABLET-VIEW */
+
+@media (min-width: 668px) {
+  nav a {
+    font-size: 20px;
+    font-weight: 600;
+    margin-right: 20px;
+  }
+
+  .head-container {
```
Comment (Rosanna86) [inline code/style.css:246]: **iPad portrait mode** (opened it with responsinator.com)
When the page is open in iPad portrait mode the header container takes up almost 80% off the screen. I understand that the links in the menu will take you to the forms. But I think when it comes to user experience maybe have a nice header/hero and have the forms only connected to links in navbar (not shown on start page). Or put forms next to each other on main page.

Screenshot 2021-09-07 at 10 19 46

**Review by Rosanna86 — COMMENTED** [review]
[YES ] Am I able to understand the code easily?
[YES ] Well-written code?
[WORKS] Does it work or does anything look broken?
[YES] Does it follow the general and blue requirements?

File: code/style.css [hunk 5]
```diff
@@ -1,7 +1,291 @@
...
+  height: 50px;
+  width: auto;
+  margin-left: 3px;
+}
+
+.hidden-anchor {
+  height: 50px;
+}
+
+.head-container {
```
[the remark this answers is not in the document]
Comment (nehrwein) [reply code/style.css:48]: I agree, would be nice to bring it in a real flow.

File: README.md [hunk 6]
```diff
@@ -1,12 +1,19 @@
...
+I started with the mobile-site. 
+- I used a hero-image in the background for the header in respect of data usage.
+- I build and styled a form with 4 different inputs (text, number, email, radio button) and a submit button
+- I build a FAQ-accordion with basic js (document.getElementbyID and then toggling to .active). The FAQ opens and closes by clicking it and is showing +/- if closed/open.
+
+On tablet-size the hero-image changes to a video
+
+On desktop-size the form and the accordion are hidden at first, but get displayed on top of the hero video after clicking the links to introduction and FAQ in the header. This works by selecting the screensize with js and then display or hide different sections of the site.
+
+ I am quite proud about what I achieved this week with using js first the first time. :)
```
[the remark this answers is not in the document]
Comment (nehrwein) [reply README.md:16]: Thank you 🙂

File: code/style.css (hunk 5 above)
[the remark this answers is not in the document]
Comment (nehrwein) [reply code/style.css:48]: Good point! I was more concerned with the look and functionality of the site and didn't think about the user experience too much. But it's actually super important.

File: code/style.css [hunk 7]
```diff
@@ -1,7 +1,291 @@
...
+  width: 100%;
+  background-color: #297F55;
+  z-index: 1;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+nav a {
+  color: yellow;
```
[the remark this answers is not in the document]
Comment (nehrwein) [reply code/style.css:31]: ;)

File: code/script.js [hunk 8]
```diff
@@ -2,8 +2,78 @@
...
+          const formSection = document.getElementById('form-section-id')
+          formSection.style.display = 'none'  
+
+  })
+} 
+
+
+
+
+
```
[the remark this answers is not in the document]
Comment (nehrwein) [reply code/script.js:68]: Yes, that does the trick!

File: code/style.css (hunk 4 above)
Comment (nehrwein) [reply code/style.css:246]: yeah, the tablet solution actually sucks 😆 I was starting with mobile and then I came up with the desktop, but I kind of neglected the tablet in between. Would have been better to either give it the desktop look as well or make something more appropriate for that screen size.
I think this is a good example of how helpful it is, when someboday else has a look at your design and points out (obvious) flaws to you. I couldn't see it before as I was so stuck in my code and the design I applied.

**Pull request closed without merging**